Dockerfile

```diff
@@ -1,35 +1,32 @@
-FROM node:14 AS builder
+FROM node:18 AS node
 
-RUN apt-get update && apt-get install -y \
-    php \
-    php-cli \
-    php-zip \
-    php-xml \
-    php-mbstring \
-    php-curl \
-    php-sqlite3 \
-    php-ldap \
-    unzip \
-    cron
+WORKDIR /build
 
-RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
-WORKDIR /app
+COPY www/package*.json .
 
-COPY . .
+RUN npm i
 
-RUN cd /app/www && npm install
-RUN cd /app/www && composer install
+# ---
 
-FROM php:8.0-apache
+FROM composer:latest AS composer
+FROM dunglas/frankenphp
```

Comment:
Why not use official PHP image?

Reply:
The official PHP image with Apache has a lot of vulnerabilities according to the generated report, so I swapped it for this image.